### ISSUE TITLE
I have encountered a error in model training and the error is  _main__:79: RuntimeWarning: overflow encountered in multiply __main__:80: RuntimeWarning: overflow encountered in multiply __main__:80: RuntimeWarning: invalid value encountered in subtract Iteration: 10 ; error = nan Iteration: 20 ; error = nan

### ISSUE THREAD

**Issue by YarraguntlaNitish** [albertauyeung/matrix-factorization-in-python#5]

**Comment by YarraguntlaNitish** [comment]
__main__:79: RuntimeWarning: overflow encountered in multiply
__main__:80: RuntimeWarning: overflow encountered in multiply
__main__:80: RuntimeWarning: invalid value encountered in subtract
Iteration: 10 ; error = nan
Iteration: 20 ; error = nan

and my training_process shows all nan's

**Comment by YarraguntlaNitish** [comment]
The problem has been solved as i reduced the alpha value and increased the number of epochs for gradient descent to converge.

Thanks
